src/Microsoft.Health.Dicom.SqlServer/Features/Store/SqlIndexDataStoreV57.cs

```diff
@@ -0,0 +1,47 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Data.SqlClient;
+using Microsoft.Health.Dicom.Core.Exceptions;
+using Microsoft.Health.Dicom.Core.Features.Common;
+using Microsoft.Health.Dicom.SqlServer.Features.Schema;
+using Microsoft.Health.Dicom.SqlServer.Features.Schema.Model;
+using Microsoft.Health.SqlServer.Features.Client;
+
+namespace Microsoft.Health.Dicom.SqlServer.Features.Store;
+
+internal class SqlIndexDataStoreV57 : SqlIndexDataStoreV55
+{
+    public SqlIndexDataStoreV57(SqlConnectionWrapperFactory sqlConnectionWrapperFactory)
+        : base(sqlConnectionWrapperFactory)
+    {
+    }
+
+    public override SchemaVersion Version => SchemaVersion.V57;
+
+    public override async Task<IndexedFileProperties> GetIndexedFileMetricsAsync(CancellationToken cancellationToken = default)
+    {
+        using SqlConnectionWrapper sqlConnectionWrapper = await SqlConnectionWrapperFactory.ObtainSqlConnectionWrapperAsync(cancellationToken, true);
+        using SqlCommandWrapper sqlCommandWrapper = sqlConnectionWrapper.CreateRetrySqlCommand();
+        VLatest.GetTotalAndSumContentLengthIndexedAsyncV57.PopulateCommand(sqlCommandWrapper);
+
+        try
+        {
+            using SqlDataReader sqlDataReader = await sqlCommandWrapper.ExecuteReaderAsync(cancellationToken);
+            await sqlDataReader.ReadAsync(cancellationToken);
+            return new IndexedFileProperties
+            {
+                TotalIndexed = (int)sqlDataReader[0],
```

Comment:
There is a chance of failure if there are no rows in the table.
Better would be 

```
while (await reader.ReadAsync(cancellationToken))
{
    return reader.ReadRow(
        TotalIndexedColumn,
        TotalSumColumn);
}
```

Reply:
You can declare the custom column like

```
    protected static readonly Health.SqlServer.Features.Schema.Model.BigIntColumn TotalIndexedColumn =
        new Health.SqlServer.Features.Schema.Model.BigIntColumn("TotalIndexed");
```